framework/src/controller/helpers/validator.js

```diff
@@ -1,7 +1,12 @@
 const Ajv = require('ajv');
 const { SchemaValidationError } = require('../../errors');
 
+const zSchema = require('../../modules/chain/helpers/z_schema');
```

Comment:
we use zSchema and Ajv...?

Reply:
@shuse2 For now, we are using both yes, as z-schema allows to define custom validators and we can load those as a format with Ajv. In the future, this will be modified to fit one, this has been discussed and decided to do it this way. Please approve if the code looks good now.